application/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "logging": {
+    "filepath": "./rd-log.log"
```

Comment:
Could we, at the very simplest level, create this config with logger names, so we pass the logger the required name?